src/containers/Bookings.js

```diff
@@ -2,28 +2,89 @@ import React, { Component } from 'react';
 import Search from '../components/Search.js';
 import Results from '../components/Results.js';
 import fakeBookings from '../data/fakeBookings.json';
+import Counter from '../components/Counter';
+//import addClickHandler from '../component/addClickHandler';
 
 export default class Bookings extends Component {
+  state = {
+    results: fakeBookings,
+    Counter:Results,
+    customerIdInput: "",
+    customerNameInput:"",
+   
+   
+  }
+
+  customerInputChange = (event) => {
+    this.setState({
+      customerIdInput: event.target.value
+    })
+  }
+  customerNameInputChange = (event) => {
+    this.setState({
+      customerNameInput: event.target.value
+    })
+  }
 
-  // state = { results: [] };
-  search = () => {
-    console.info('TO DO!');
-    // this.setState({ results: fakeBookings });
-  };
+  
+  
+ 
 
+  filteredCustomerId = (event) => {
+    const { customerIdInput, results } = this.state;
+    // const customerIdInput = this.state.customerIdInput;
+    // const results = this.state.results;
+
+    this.setState({
+      results: results.filter(customer => customer.id === parseInt(customerIdInput, 10))
```

Comment:
Good usage of `parseInt` 👍 And adding the second argument because that is always necessary in JS.